File: test/language/expressions/java/expressions/ValueOfOperatorTest.java
package expressions;

import static org.junit.jupiter.api.Assertions.*;
import org.junit.jupiter.api.Test;

import expressions.valueof_operator.ValueOfFunctions;
import expressions.valueof_operator.Color;

public class ValueOfOperatorTest
{
    @Test
    public void getValueOfWhiteColor()
    {
        final ValueOfFunctions valueOfFunctions = new ValueOfFunctions(Color.WHITE);
        final short whiteColorValue = 1;
        assertEquals(whiteColorValue, valueOfFunctions.funcGetValueOfColor());
        assertEquals(whiteColorValue, valueOfFunctions.funcGetValueOfWhiteColor());
    }

    @Test
    public void getValueOfBlackColor()
    {
        final ValueOfFunctions valueOfFunctions = new ValueOfFunctions(Color.BLACK);
        final short blackColorValue = 2;
        assertEquals(blackColorValue, valueOfFunctions.funcGetValueOfColor());
        assertEquals(blackColorValue, valueOfFunctions.funcGetValueOfBlackColor());
    }
}

File: test/extensions/language/expressions/python/ValueOfOperatorTest.py
import Expressions


class ValueOfOperatorTest(Expressions.TestCase):
    def testGetValueOfWhiteColor(self):
        valueOfFunctions = self.api.ValueOfFunctions(self.api.Color.WHITE)
        whiteColorValue = 1
        self.assertEqual(whiteColorValue, valueOfFunctions.get_value_of_color())
        self.assertEqual(whiteColorValue, valueOfFunctions.get_value_of_white_color())

    def testGetValueOfBlackColor(self):
        valueOfFunctions = self.api.ValueOfFunctions(self.api.Color.BLACK)
        blackColorValue = 2
        self.assertEqual(blackColorValue, valueOfFunctions.get_value_of_color())
        self.assertEqual(blackColorValue, valueOfFunctions.get_value_of_black_color())
